dev_utils/compose-sda.yml

```diff
@@ -48,6 +48,19 @@ services:
       - certs:/dev_utils/certs
     mem_limit: 64m
     restart: always
+  sync:
+    command: sda-sync
+    container_name: sync
+    depends_on: 
+      - certfixer
+    env_file: ./env.sync
+    image: neicnordic/sda-pipeline:latest
+    volumes:
+      - ./config.yaml:/config.yaml
+      - ./:/dev_utils/
+      - certs:/dev_utils/certs
+    mem_limit: 64m
+    restart: always
```

Comment:
Wont this create a round robin scenario since both finalize and sync reacts to the same messages?

Reply:
But aren't we supposed to have only sync or finalize, but not both?

Reply:
Yes, but what you have here will start both if you do `docker-compose up`.
It will work but we need to document that the unwanted service needs to be shut down before any tests are run.

Reply:
Better to have them use different queues and configure `verify` to send to the desired one?

Reply:
(From principle, while it's quite minimal, I'd rather prefer not having the functionality duplicated but rather have `sync` act as a pass-through, sending onwards to `finalize` once the copy is done. Both alternatives obviously works, though.)